docs/MigrationGuides/migration_9_to_10.md

```diff
@@ -0,0 +1,21 @@
+# Migration Guide
+
+This project follows [Semantic Versioning](http://semver.org).
+
+Please follow the appropriate guide below when **upgrading to a new major version** of Moya (e.g. 9.0 -> 10.0).
```

Comment:
This line is probably not needed now as we have only one migration guide per file.  Maybe also let's merge the titles  `Migration Guide` and `Upgrade from 9.x to 10.x` to something like `Migration guide from 9.x to 10.x`? What do you think?

Reply:
Yeah I agree 👍 I'll make these changes